main/src/main/scala/sbt/plugins/SemanticdbPlugin.scala

```diff
@@ -21,14 +21,14 @@ object SemanticdbPlugin extends AutoPlugin {
     semanticdbEnabled := false,
     semanticdbIncludeInJar := false,
     semanticdbOptions := List("-Yrangepos"),
-    semanticdbVersion := "4.1.0",
-    semanticdbCompilerPlugin := {
-      val v = semanticdbVersion.value
-      ("org.scalameta" % "semanticdb-scalac" % v).cross(CrossVersion.full)
-    }
+    semanticdbVersion := "4.1.0"
```

Comment:
wondering if we could bump this

Reply:
Yes please - https://github.com/sbt/sbt/issues/4633

Reply:
Done 👍